boards/common/particle-mesh/Makefile.include

```diff
@@ -0,0 +1,24 @@
+export CPU_MODEL = nrf52840xxaa
+
+# set default port depending on operating system
+PORT_LINUX ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+
+# add the common header files to the include path
+INCLUDES += -I$(RIOTBOARD)/common/particle-mesh/include
+
+# This board uses a DAP-Link programmer
+# Flashing support is provided through pyocd (default) and openocd.
+# For openocd, a version built against the development branch and containing
+# the support for nrf52 cpu is required.
+PROGRAMMER ?= pyocd
+ifeq (pyocd,$(PROGRAMMER))
+  # The board is not recognized automatically by pyocd, so the CPU target
+  # option is passed explicitly
+  export FLASH_TARGET_TYPE ?= -t nrf52840
```

Comment:
This is not working for when a stlink is connected on a different port, is this expected behaviour?